sources/packages/web/src/components/common/Reports.vue

```diff
@@ -62,8 +62,20 @@ export default defineComponent({
       );
     };
     const formChanged = async (form: FormIOForm, event: FormIOChangeEvent) => {
-      // Populates the program year select component if required.
-      if (event.changed.component.key === REPORT_TYPE_DROPDOWN_KEY) {
+      // Populates the program year select component, institution select component if required.
+      if (event.changed?.component.key === REPORT_TYPE_DROPDOWN_KEY) {
+        if (event.changed["value"] === "Ministry_Student_Unmet_Need_Report") {
+          form["root"].getComponent(
+            "institutionId",
+          ).component.validate.required = false;
+          form["root"].getComponent("institutionId").triggerRedraw();
+        }
+        if (event.changed["value"] === "Program_And_Offering_Status_Report") {
+          form["root"].getComponent(
+            "institutionId",
+          ).component.validate.required = true;
+          form["root"].getComponent("institutionId").triggerRedraw();
+        }
```

Comment:
Please type the properties and create the helpers in `useFormIOUtils` as discussed.